superset/assets/package.json

```diff
@@ -89,7 +89,7 @@
     "react-datetime": "2.9.0",
     "react-dom": "^15.6.2",
     "react-gravatar": "^2.6.1",
-    "react-grid-layout": "^0.16.0",
+    "react-grid-layout": "0.16.5",
```

Comment:
Can you add a comment on the following line? Since JSON doesn't support comments, the best way might be to:
`"//": "react-grid-layout 0.16.6 has know issue described in #4596"`

Reply:
hopefully `yarn` and `npm` won't try to reorder the comment. Related: https://stackoverflow.com/questions/14221579/how-do-i-add-comments-to-package-json-for-npm-install